gamemodes/terrortown/gamemode/client/vgui/cl_sb_team.lua

```diff
@@ -123,51 +131,62 @@ function PANEL:HasRows()
 	return self.rowcount > 0
 end
 
-function PANEL:UpdateSortCache()
-	self.rows_sorted = {}
-
-	for _, row in pairs(self.rows) do
-		table.insert(self.rows_sorted, row)
+local function SortFunc(rowa, rowb)
+	if not IsValid(rowa) or not IsValid(rowb) then
+		return false
```

Comment:
If only rowb is not valid, the sort function should return true. A valid entry should always be above an invalid entry.

Reply:
That gives a wrong bot name sorting

Reply:
This sounds strange. In which case should an invalid row be in front of a valid one?